### ISSUE TITLE
Can't delete a file

### ISSUE THREAD

**Issue by k-saari** [globaleaks/GlobaLeaks#2995]
Can't delete files what i have upload to globaleaks. After clicking Delete button, it returns to the Main configuration tab but not delete file.

**Comment by evilaliv3** [comment]
Thank you for reporting this.

This issue should have been fixed in release 4.2.3

Which is the version that are you currently using?

**Comment by k-saari** [comment]
First I tried to upgrade our globaleaks. But although the changelog shows that Changes in version 4.2.3 but still our software versio is 4.2.2. 2 hours ago I tried to install a very clean installation on a virtual machine because I thought I had done something but the problem didn't go away still same problem.

Now I was trying to update this version what is on my virtual machine it says I already have the latest version 4.2.3. Although when I looking at changelog it says software version 4.2.2.
